### ISSUE TITLE
ScoreTiler improvement

### ISSUE THREAD

**Issue by ernestoarbitrio** [histolab/histolab#253]
- Scorer Locate
- Heatmap

**Comment by alessiamarcolini** [comment]
Yes the idea is to have a method for the `ScoreTiler` to output an image like this, showing an heatmap of the scores on the tiles
![score-tiler-paper](https://user-images.githubusercontent.com/20052362/115149882-59490c80-a066-11eb-8353-e0b6e7e6a8b1.png)
